html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/Authentication/Password.pm

```diff
@@ -0,0 +1,87 @@
+package captiveportal::PacketFence::DynamicRouting::Module::Authentication::Password;
+
+=head1 NAME
+
+captiveportal::DynamicRouting::Module::Authentication::Password
+
+=head1 DESCRIPTION
+
+Login registration
```

Comment:
Add the same description as in the extension point, its clearer:
`Login Controller with a predifined username for captiveportal`